packages/client/hmi-client/src/components/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss-drilldown.vue

```diff
@@ -719,7 +719,8 @@ watch(
 	async () => {
 		// Update selected output
 		if (props.node.active) {
-			selectedOutputId.value = props.node.active;
+			const active = props.node.active;
+			selectedOutputId.value = props.node.outputs.find((o) => o.id === active)?.value?.[0];
```

Comment:
A bit confused - selectedOutputId doesn't seem to be used in this component, it is declared at the top and assigned here, so how is it causing an issue?

Reply:
Apologizes this is not required here!